frontend/packages/dev-console/src/components/import/ImportForm.tsx

```diff
@@ -153,6 +162,14 @@ const ImportForm: React.FC<ImportFormProps & StateProps> = ({
     },
     healthChecks: healthChecksProbeInitialData,
   };
+  const [initialValuesState, setInitialValuesState] = React.useState(initialValues);
+  React.useEffect(() => {
+    if (knScalingConfig) {
+      setInitialValuesState((iv) => {
+        return setKnScalingDefaultValue(iv, knScalingConfig);
+      });
+    }
+  }, [knScalingConfig]);
```

Comment:
as it's used in `DeployImage.tsx ` as well , will it make sense to abstract as custom hook?